internal/database/database_container.go

```
@@ -0,0 +1,388 @@
+	// Wait till the database server is responding:
+	url := fmt.Sprintf(
+		"postgres://postgres:%s@%s:%s/postgres?sslmode=disable",
+		password, host, port,
+	)
+	handle, err := sql.Open("pgx", url)
+	if err != nil {
+		return fmt.Errorf("failed to open database connection: %w", err)
+	}
+	deadline := time.Now().Add(30 * time.Second)
+	for {
+		err = handle.Ping()
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Ping must not be called. use (*database/sql.DB).PingContext

(noctx)